lib/movie.rb

```diff
@@ -0,0 +1,18 @@
+require './item'
+
+# rubocop:disable Metrics/ParameterLists
+class Movie < Item
+  def initialize(genre, author, source, label, publish_date, silent)
+    super(genre, author, source, label, publish_date)
+    @silent = silent
+  end
+  # rubocop:enable Metrics/ParameterLists
+
+  private
+
+  def can_be_archived?
+    return true if super || @silent
+
+    false
```

Comment:
[Optional] suggestions:
Just a suggestion, no need to use false here as the method will return false automatically if the true condition isn't met

Reply:
Thanks Fabien. You are right.